plugins/HAA4bAnalysis.cc

```diff
@@ -118,9 +135,18 @@ class HAA4bAnalysis : public edm::EDAnalyzer {
   float _Nevents_mpairs;
   float _Nevents_deltaM;
   float _Nevents_passed;
+  unsigned int _nPv; 
+
+  //Few new counters and variables
+  float npT;
+  float npIT;
+  double MyWeight;
```

Comment:
Can you call MyWeight something like PU_weight or similar? We are going to add other weights and it's better to have clear names for each one